### PR TITLE
4412 Update Forms

### PR DESCRIPTION
**Pull request by Liliya-web**

### CODE REVIEW

File: components/form_inputs/forms/DisabledForm.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+</template>
+<script>
+export default {
+  data: () => ({
+    allowSpaces: false,
+    match: 'Foobar',
+    max: 0,
+    model: 'Foobar',
+  }),
+  computed: {
```
Comment (omega790) [inline components/form_inputs/forms/DisabledForm.vue:37]: Do we really need this for this form?

File: components/form_inputs/forms/ReadonlyForm.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+</template>
+<script>
+export default {
+  data: () => ({
+    allowSpaces: false,
+    match: 'Foobar',
+    max: 0,
+    model: 'Foobar',
+  }),
+  computed: {
```
Comment (omega790) [inline components/form_inputs/forms/ReadonlyForm.vue:37]: Do we really need this for this form?

File: components/form_inputs/forms/DisabledForm.vue (hunk 1 above)
Comment (Liliya-web) [reply components/form_inputs/forms/DisabledForm.vue:37]: Why not? To check disabled property

File: components/form_inputs/forms/ReadonlyForm.vue (hunk 2 above)
Comment (Liliya-web) [reply components/form_inputs/forms/ReadonlyForm.vue:37]: Yes. To check readonly property

File: components/form_inputs/forms/DisabledForm.vue (hunk 1 above)
Comment (omega790) [reply components/form_inputs/forms/DisabledForm.vue:37]: I am talking about script section, it's about rules to check input, but we have disabled or readonly form so we can't type something

Comment (Liliya-web) [reply components/form_inputs/forms/DisabledForm.vue]: Fixed

File: components/form_inputs/forms/ReadonlyForm.vue (hunk 2 above)
Comment (Liliya-web) [reply components/form_inputs/forms/ReadonlyForm.vue]: Fixed

**Review by omega790 — APPROVED** [review]

**Pull request merged**